### PR TITLE
Add Qitmeer chain

### PR DESCRIPTION
**Pull request by fiboswap**
Add Qitmeer chain

Fixes # .

Changes proposed in this pull request:
- Add support for Qitmeer EVM mainnet networks
- 
-

### CODE REVIEW

**Comment by fiboswap** [comment]
How to complete the merging as soon as possible?

**Review by Heheehd — APPROVED** [review]

**Review by Heheehd — APPROVED** [review]

File: src/networks.json [hunk 1]
```diff
@@ -1848,5 +1848,21 @@
...
     "start": 2560501,
     "logo": "ipfs://QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8"
+  },
+  "813": {
+    "key": "813",
+    "name": "Qitmeer Mainnet",
+    "shortName": "MEER",
+    "chainId": 813,
+    "network": "mainnet",
+    "multicall": "0x55034b2cF530ae3A8fC1e2e4523F58496796610F",
```
Comment (ChaituVR) [inline src/networks.json:1858]: @fiboswap https://evm.meerscan.io/address/0x55034b2cF530ae3A8fC1e2e4523F58496796610F/contracts#address-tabs

Can verify and publish multicall code here?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Everything else looks good

File: src/networks.json (hunk 1 above)
Comment (fiboswap) [reply src/networks.json:1858]: Verified

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**